packages/arakoon/arakoon.1.8.11/opam

```diff
@@ -0,0 +1,43 @@
+opam-version: "1.2"
+maintainer: "Romain Slootmaekers <romain.slootmaekers@openvstorage.com>"
+authors: ["Romain Slootmaekers <romain.slootmaekers@openvstorage.com>"
+          "Jan Doms <jan.doms@openvstorage.com>"]
+homepage: "https://github.com/openvstorage/arakoon"
+bug-reports: "https://github.com/openvstorage/arakoon"
```

Comment:
This could end in `/issues` to be precise but I think this is close enough to merge.